libpay-stripe/src/main/scala/com/wix/pay/stripe/StripeGateway.scala

```diff
@@ -48,8 +52,17 @@ class StripeGateway(merchantParser: StripeMerchantParser = new JsonStripeMerchan
     Token.create(params, requestOptionsFor(apiKey))
   }
 
-  private def requestOptionsFor(apiKey: String): RequestOptions = {
-    RequestOptions.builder.setApiKey(apiKey).build
+  private def requestOptionsFor(apiKey: String): RequestOptions = RequestOptions.builder
+    .setApiKey(apiKey)
+    .setWith(_.setConnectTimeout, connectTimeout)
+    .setWith(_.setReadTimeout, readTimeout)
+    .build
+
+  private implicit class `RequestOptionsTimeoutBuilder`(builder: RequestOptionsBuilder) {
```

Comment:
`RequestOptionsTimeoutBuilder`, `maybeTimeout` and `timeout` should be renamed I think :)